captum/insights/features.py

```diff
@@ -116,7 +116,11 @@ def visualize(self, attribution, data, contribution_frac) -> FeatureOutput:
         attribution = attribution.sum(dim=1)
 
         # L-Infinity norm
-        normalized_attribution = attribution / abs(attribution).max()
+        normalized_attribution = attribution
+        attr_max = abs(attribution).max()
+        if attr_max > 0:
```

Comment:
Probably want to refactor this into a function, since I do it multiple times (I think it is also done [here](https://github.com/pytorch/captum/blob/master/captum/insights/api.py#L154)) - just not sure where I should place it